ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-release-4.20__periodics.yaml

```diff
@@ -89,27 +89,53 @@ tests:
     - ref: baremetalds-e2e-ovn-bgp-pre
     workflow: baremetalds-e2e-ovn-dualstack
   timeout: 5h0m0s
-- as: e2e-aws-kubevirt-ovn-techpreview
+- as: e2e-metal-ipi-ovn-virt-dualstack
   cluster: build05
-  cron: 0 0 * * *
+  cron: 0 3 * * *
   steps:
-    cluster_profile: aws
+    cluster_profile: equinix-ocp-metal
     env:
       CNV_PRERELEASE_LATEST_CHANNEL: "true"
-      COMPUTE_NODE_REPLICAS: "2"
-      COMPUTE_NODE_TYPE: c5n.metal
-      FEATURE_SET: TechPreviewNoUpgrade
-    workflow: openshift-e2e-aws-ovn-virt
-- as: e2e-aws-ovn-virt-periodic
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v4v6
+      EXTRA_MG_ARGS: --host-network
+      EXTRA_NETWORK_CONFIG: |
+        EXTRA_NETWORK_NAMES="extranet"
+        EXTRANET_NETWORK_SUBNET_V4='192.168.221.0/24'
+        EXTRANET_NETWORK_SUBNET_V6='fd2e:6f44:5dd8:ca56::/120'
+      TEST_SUITE: openshift/network/virtualization
+    post:
+    - chain: baremetalds-ofcir-post
+    pre:
+    - chain: baremetalds-ofcir-pre
+    - ref: baremetalds-devscripts-conf-extranetwork
+    - ref: kubevirt-install
+    workflow: baremetalds-e2e-ovn-dualstack
+  timeout: 6h0m0s
+- as: e2e-metal-ipi-ovn-virt-dualstack-techpreview
   cluster: build05
   cron: 0 3 * * *
```

Comment:
should we choose another cron ? i.e. run this periodic at a different time ?

Reply:
Let's start with this, then we can adapt, maybe moving to other times affects other things who knows...